roles/sro_deploy_art_bundle/tasks/main.yml

```diff
@@ -0,0 +1,190 @@
+---
+- name: Add brew pull secrets
+  block:
+    - name: Grab the current secrets
+      command: oc get secret/pull-secret -n openshift-config -o json | jq -r '.data.".dockerconfigjson"' | base64 -d > "{{ artifact_extra_logs_dir }}/authfile"
```

Comment:
this is a `shell`, not a `command`
and you'll need `set -o pipefail` to avoid (legit) `ansible-lint` complains

can you split long commands into multiple lines?
for this one I would suggest something like:
```
shell: 
  oc get secret/pull-secret -n openshift-config -o json 
      | jq -r '.data.".dockerconfigjson"' 
      | base64 -d 
      > "{{ artifact_extra_logs_dir }}/authfile"
```

Reply:
Done